src/main/logging/local_es/elasticsearch.yaml

```diff
@@ -0,0 +1,261 @@
+---
```

Comment:
Is it necessary to define all the parameters? Can we rely on defaults and just override the ones that are required?

Reply:
we can do not provide any value file at al1, because all the changes are related to the resource management and there is no specific change which rely to specific deployment itself. 
However, having this complete file gives the better sight to client about their options specially when we are not officially support.